metadata/boundaries/usfs_roadless_inventory.md

```diff
@@ -0,0 +1,73 @@
+# Title
+
+Utah USFS Roadless Inventory
+
+## ID
+
+ae3e16a4-13a8-4af6-a977-b1989ad9880a
+
+## Brief Summary
+
+Polygon dataset of areas in Utah designated as roadless by the 2000 USFS Roadless Inventory.
```

Comment:
```suggestion
Polygon dataset of areas in Utah designated as roadless by the 2001 USFS Roadless Inventory.
```
It looks like this is technically 2001: https://www.fs.usda.gov/detail/roadless/2001roadlessrule/maps/?cid=stelprdb5382437